lib/index.js

```diff
@@ -3,9 +3,4 @@
 var Server = require('./server');
 
 
-// Declare internals
```

Comment:
Can you point me to where that is called out in the style guide?  I don't see it.